### PR TITLE
Add OpenTelemetry data in New Relic -> overview, replacing alerts and custom queries pages

### PR DESCRIPTION
**Pull request by jack-berg**
- Add new "OpenTelemetry -> OpenTelemetry data in New Relic -> Overview" page
- Remove "OpenTelemetry -> OpenTelemetry data in New Relic -> Alerts" page, replacing with link to general alerts page on the new overview page
- Remove "OpenTelemetry -> OpenTelemetry data in New Relic -> Create custom queries" page, replacing with link to general querying page on the new overview page

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* [OpenTelemetry traces in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-traces/)
+* [OpenTelemetry metrics in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics/)
+* [OpenTelemetry logs in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs/)
+* [OpenTelemetry resources in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources/)
+
+All OTLP data is ingested into the [New Relic platform](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform/), which includes many powerful general purpose observability tools to help you identify and diagnose problems, including:
+
+* [Alerts](/docs/alerts-applied-intelligence/overview/): get informed when a signal exceeds a threshold.
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:28]:
```suggestion
* [Alerts](/docs/alerts-applied-intelligence/overview/): Get informed when a signal exceeds a threshold.
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* [Custom Queries](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/): write custom NRQL queries for adhoc data analysis and exploration.
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:29]:
```suggestion
* [Custom Queries](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/): Write custom NRQL queries for adhoc data analysis and exploration.
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* [Dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/): create custom data visualizations reflecting the unique requirements of your applications.
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:30]:
```suggestion
* [Dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/): Create custom data visualizations reflecting the unique requirements of your applications.
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-data-explorer-query-builder/
+---
+
+OpenTelemetry has a rich ecosystem of instrumentation and tooling which can be integrated with New Relic via [OTLP](https://opentelemetry.io/docs/specs/otlp/). This documentation describes how New Relic receives, processes, and ingests OTLP data. See [Get started with OpenTelemetry and New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro/) for information on common integration patterns.
+
+[New Relic OTLP endpoint](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) describes OTLP configuration requirements relevant for any OpenTelemetry integration with New Relic, along with [troubleshooting](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp-troubleshooting/) information.
+
+OTLP is divided into trace, metric, and log signals, all of which include the concept of a resources. See the following pages for details on how each is processed and ingested:
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:19]:
```suggestion
OTLP is divided into trace, metric, and log signals, all of which include the concept of resources. See the following pages for details on how each is processed and ingested:
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+title: OpenTelemetry data in New relic
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+metaDescription: Overview of OpenTelemetry data in New RElic
+freshnessValidatedDate: never
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:9]:
```suggestion
freshnessValidatedDate: 2024-05-31
```

**Review by rhetoric101 — COMMENTED** [review]
@jack-berg thanks for adding this new overview page and for removing old content! I just made a few minor suggestions. Also, could we remove ": Best practices" in the title for the "OpenTelemetry resources" page? I think that "Best practices" is a carryover from a previous pattern when we had those types of pages.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx (hunk 1 above)
Comment (ally-sassman) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx:28]: @Rob-S , we already spoke about this issue and have decided to hold off on changing the URLs until the final yml structure has been confirmed.

**Review by ghost — APPROVED** [review]

**Comment by ally-sassman** [comment]
@jack-berg Can this be merged?

**Comment by jack-berg** [comment]
Yes @ally-sassman! Thanks.

**Pull request merged**